### ISSUE TITLE
Support recurring contributions

### ISSUE THREAD

**Issue by twomice** [progressivetech/net.ourpowerbase.remoteform#26]
Is the project interested in adding (full or partial) support for recurring contributions? If so, we're interested in working on a PR to help make that happen. I don't have a lot of details to offer at this point, just general interest and willingness to a) be talked out of it, and/or b) submit a quality PR that would actually be useful.

Any thoughts on that?

**Comment by jmcclelland** [comment]
The quick answer is: yes! I love it. Supporting recurring contributions would be great.

The long answer is... this project is in a bit of a holding pattern while we watch both APIv4 (well, not so much any more, APIv4 is pretty stable now) and Afform.

I know that this project will eventually only suppport afform - so you build a form in Civi using Afform, then you link it up with remote form and you are done.

What I don't know is when to re-write this code to support afform.

Here are some scenarios:

1. Don't do any more work on this version. Start working on an afform version. 

2. Continue adding new features to this version and then re-implement then later with afform.

3. Rewrite this version to use apiv4 (maybe that will help prepare for afform??)

Geez, this is a really unhelpful answer to your very simple question, but I hope it gives you a clue as to why I'm a bit paralyzed.

**Comment by twomice** [comment]
My client has some time pressures that incline us to aim for a PR against the current state of the extension; I'm open to finding ways to make that work as useful as possible to the overall project and its various possible futures. If you have thoughts on that, please let me know. In the meantime, we may just jump in and see what happens.

**Comment by jmcclelland** [comment]
Yes - that sounds great. I'd welcome a PR with the caveat that this whole code base may get abandoned soon (but also with the caveat on the caveat that this feature would be fully welcomed in the new version as well :)

**Comment by twomice** [comment]
PR #37 . (Not sure why github didn't pick it up as usual.)
